lib/new_relic/harvest/telemetry_sdk/api.ex

```diff
@@ -17,6 +17,14 @@ defmodule NewRelic.Harvest.TelemetrySdk.API do
     |> maybe_retry(url, payload)
   end
 
+  def dimensional_metric(metrics) do
+    url = url(:metric)
+    payload = {:metrics, metrics, generate_request_id()}
+
+    request(url, payload)
```

Comment:
We may want to review the status codes this module knows about against the status codes the dimensional metrics API responds with:

https://docs.newrelic.com/docs/data-apis/ingest-apis/metric-api/report-metrics-metric-api/#response-status-codes

(I haven't, and maybe there's nothing to worry about)